draft-irtf-cfrg-vdaf.md

```diff
@@ -3977,13 +3977,41 @@ parameter with the same level has been used with the same input share before.
 The whole preparation phase MUST NOT be run more than once for a given
 combination of input share and level.
 
+<!-- TODO: Text about counts -->
```

Comment:
Here we should list the invariants we should expect to hold for each aggregation.

WDYTA enforcing that the sum of the prefix counts is equal to the number of valid reports in the batch?

Reply:
Added some text.

> WDYTA enforcing that the sum of the prefix counts is equal to the number of valid reports in the batch?

This is enforced in `aggregate`.